CHANGELOG.md

```diff
@@ -1,5 +1,57 @@
 # Release Notes
 
+## Version 1.27.1 - 2022-10-07
+
+Features (Experimental):
+* Implement socket_vmnet network (QEMU) for macOS [#14989](https://github.com/kubernetes/minikube/pull/14989)
```

Comment:
Lets reword
"Qemu Driver: Add support for dedicated network on MacOs (socket_vmnet)"
in the second line (maybe same PR link)
"Qemu Driver: Add support minikube service and tunnel on MacOs"

Reply:
Reworded